content/docs/about/style-guide.md

```diff
@@ -0,0 +1,323 @@
+          the Internet.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+**Exception:** Use future tense if it's necessary to convey the correct
+meaning. This requirement is rare.
+
+## Address the audience directly
+
+Using "we" in a sentence can be confusing, because the reader may not know
+whether they're part of the "we" you're describing. For example, compare the
+following two statements:
+
+* "In this release we've added many new features."
+* "In this tutorial we build a flying saucer."
+
+The words "the developer" or "the user" can be ambiguous. For example, if the
+reader is building a product that also has users, then the reader does not
+know whether you're referring to the reader or the users of their product.
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Address the reader directly</th>
+        <th>Instead of "we", "the user", or "the developer"</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Include the directory in your path</td>
+        <td>The user must make sure that the directory is included in their path.
+        </td>
+      </tr>
+      <tr>
+        <td>In this tutorial you build a flying saucer</td>
+        <td>In this tutorial we build a flying saucer</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Use short, simple sentences
+
+Keep sentences short. Short sentences are easier to read than long ones.
+Below are some tips for writing short sentences.
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th colspan="2">Use fewer words instead of many words that convey the same meaning</th>
+      </tr>
+      <tr>
+        <th>Use this</th>
+        <th>Instead of this</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>You can use</td>
+        <td>It is also possible to use</td>
+      </tr>
+      <tr>
+        <td>You can</td>
+        <td>You are able to</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th colspan="2">Split a single long sentence into two or more shorter ones</th>
+      </tr>
+      <tr>
+        <th>Use this</th>
+        <th>Instead of this</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>You do not need a running GKE cluster. The deployment process
+          creates a cluster for you.</td>
+        <td>You do not need a running GKE cluster, because the deployment 
+          process creates a cluster for you.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th colspan="2">Convert a long sentence showing various options into a bulleted list</th>
```

Comment:
In the example below, the order is important. Should this example describe when to use an ordered list and when to use a bulleted list?

Reply:
That's a good point. At this point, I'm thinking the distinction between a bulleted list and a numbered list is a little too much detail for this guide (though I'm happy to be persuaded otherwise) and if we do include that distinction, it'd need to be in a separate section about lists rather than about shortening long sentences. I've converted this example into a numbered list.